CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Moved changelog validation up in CI pipeline
 
+## [1.3.8] - 2019-04-05
```

Comment:
Unless we're bumping the version also (and I don't think we necessarily need to) this line is not needed. also, this bullet can go with the other `Changed` item